components/ProjectList/index.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import styled from "styled-components";
+import Card from "../Card";
+import useSWR from "swr";
+
+export default function ProjectList() {
+  const {
+    data: projects,
+    isLoading,
+    error,
+  } = useSWR("/api/projects", {
+    fallbackData: [],
+  });
+
+  if (error) return <p>An error has occurred.</p>;
+  if (!projects || isLoading) return <p>Projects are loading</p>;
```

Comment:
I'd strongly advise against this pattern. If you ever need the data returned by your database anywhere else, you'll have to keep fetching it. I'd always fetch my data at the top level of my app (here: `_app.js`).

Reply:
thank you, we changed that :)